### PR TITLE
WynntilsQuestBookScreen: Ability to see mini-quests

### PR DESCRIPTION
**Pull request by kristofbolyai**
I accidentally pushed https://github.com/Wynntils/Artemis/commit/f310268fb7ddde5e75d4336d5756d54f7575a156 to main. That commit belongs here. 

image

### CODE REVIEW

**Comment by byBackfish** [comment]
Looks good in-game 👍

**Review by byBackfish — DISMISSED** [review]

**Comment by magicus** [comment]
From the screen shot it looks like almost all you can see is "Mini-Quest - Slay" and that's close to redundant... Maybe at least the "Mini-Quest - " prefix can be stripped off?

**Comment by kristofbolyai** [comment]
> From the screen shot it looks like almost all you can see is "Mini-Quest - Slay" and that's close to redundant... Maybe at least the "Mini-Quest - " prefix can be stripped off?

Good idea.

File: common/src/main/java/com/wynntils/gui/screens/WynntilsQuestBookScreen.java [hunk 1]
```diff
@@ -207,6 +223,22 @@ public void onQuestsReloaded(QuestBookReloadedEvent.QuestsReloaded event) {
...
     }
 
+    @SubscribeEvent
+    public void onQuestsReloaded(QuestBookReloadedEvent.MiniQuestsReloaded event) {
+        if (!miniQuestMode) return;
+
+        this.setQuests(QuestBookManager.getMiniQuests());
+
+        for (QuestInfo quest : quests) {
+            if (!quest.isTracked()) {
```
Comment (magicus) [inline common/src/main/java/com/wynntils/gui/screens/WynntilsQuestBookScreen.java:235]: Can you track mini-quests? I did not know that. I've only read the signs and then come back and clicked with the material...

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/gui/screens/WynntilsQuestBookScreen.java:235]: You can. I added the ability to track them in the quests manager so I know for sure it is possible.

File: common/src/main/resources/assets/wynntils/lang/en_us.json [hunk 2]
```diff
@@ -542,6 +542,10 @@
   "screens.wynntils.wynntilsQuestBook.mainMap.description": "Click here to view the main map.",
   "screens.wynntils.wynntilsQuestBook.mainMap.name": "Main Map",
   "screens.wynntils.wynntilsQuestBook.name": "Wynntils Quest Book",
+  "screens.wynntils.wynntilsQuestBook.questInfo.click": "Click to see Mini-Quests",
+  "screens.wynntils.wynntilsQuestBook.questInfo.click2": "Click to see Quests",
```
Comment (magicus) [inline common/src/main/resources/assets/wynntils/lang/en_us.json]: The `2` at the end is not ideal. Can you rename it like `...questInfo.miniQuests.click` etc?

**Comment by kristofbolyai** [comment]
image

**Review by magicus — APPROVED** [review]

**Pull request merged**